editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1319,6 +1553,44 @@ SpriteFramesEditor::SpriteFramesEditor() {
 	HBoxContainer *hbc = memnew(HBoxContainer);
 	sub_vb->add_child(hbc);
 
+	playback_container = memnew(HBoxContainer);
+	hbc->add_child(playback_container);
+
+	play_bw_from = memnew(Button);
+	play_bw_from->set_flat(true);
+	play_bw_from->set_tooltip_text(TTR("Play selected animation backwards from current pos. (A)"));
+	playback_container->add_child(play_bw_from);
+
+	play_bw = memnew(Button);
+	play_bw->set_flat(true);
+	play_bw->set_tooltip_text(TTR("Play selected animation backwards from end. (Shift+A)"));
+	playback_container->add_child(play_bw);
+
+	stop = memnew(Button);
+	stop->set_flat(true);
+	stop->set_tooltip_text(TTR("Stop animation playback. (S)"));
```

Comment:
```suggestion
	stop->set_tooltip_text(TTR("Pause/stop animation playback. (S)"));
```

Or we can update the button's tooltip text. The button works correctly: the first time it is pressed it pauses, the second it stops (the frame is reset).